inttest/basic/basic_test.go

```diff
@@ -74,6 +74,7 @@ func (s *BasicSuite) TestK0sGetsUp() {
 	localPort, err := fw.LocalPort()
 	s.Require().NoError(err)
 	s.T().Log("waiting for node to be ready")
+	s.Require().NoError(util.WaitForSecret(s.Context(), kc, "kmc-test-kubeconfig", "kmc-test"))
```

Comment:
Should we just add this waiter to the `util.GetKMCClientSet` function?

Reply:
Makes sense to add it.

Reply:
done